src/ocrd_network/cli/client.py

```diff
@@ -31,56 +61,92 @@ def processing_cli():
     pass
 
 
+@processing_cli.command('check-log')
+@click.option('--address',
+              help='The address of the Processing Server. If not provided, '
+                   'the "OCRD_NETWORK_SERVER_ADDR_PROCESSING" env variable is used by default')
+@click.option('-j', '--processing-job-id', required=True)
+def check_processing_job_status(address: Optional[str], processing_job_id: str):
+    """
+    Check the log of a previously submitted processing job.
+    """
+    client = Client(server_addr_processing=address)
+    response = client.check_job_log(job_id=processing_job_id)
+    print(response._content.decode(encoding='utf-8'))
+
+
+@processing_cli.command('check-status')
+@click.option('--address',
+              help='The address of the Processing Server. If not provided, '
+                   'the "OCRD_NETWORK_SERVER_ADDR_PROCESSING" env variable is used by default')
+@click.option('-j', '--processing-job-id', required=True)
+def check_processing_job_status(address: Optional[str], processing_job_id: str):
+    """
+    Check the status of a previously submitted processing job.
+    """
+    client = Client(server_addr_processing=address)
+    job_status = client.check_job_status(processing_job_id)
+    assert job_status
+    print(f"Processing job status: {job_status}")
+
+
 @processing_cli.command('processor')
```

Comment:
Any reason why this is not called `run`, like for workflows?

Reply:
I agree – that's more consistent. (And "run ocrd-cis-ocropy-segment" sounds better than "process ocrd-cis-ocropy-segment".)

That proposal might have [originated with me](https://github.com/OCR-D/core/issues/1032#issuecomment-1511482113). If so, please feel free to change.

Reply:
Thanks for the background info. `run` would be better, but I would not want to break existing scripts right in this PR, but we could do it for 3.0 stable (with a `:fire:` entry in the CHANGELOG).

Reply:
[6608539](https://github.com/OCR-D/core/pull/1269/commits/66085392bc35334530b520b859f8760dc3518b80). 

> `run` would be better, but I would not want to break existing scripts right in this PR

In the processing server the endpoint has already been refactored to `/processor/run/{processor_name}` in the past. So not much to be changed. Only scripts using the `ocrd network client` cli would break. Me and @joschrew have none, our scripts are submitting the jobs directly to the Processing Server without the client.